config/locales/forms/check_contact_address_forms/en.yml

```diff
@@ -0,0 +1,16 @@
+en:
+  waste_exemptions_engine:
+    check_contact_address_forms:
+      new:
+        title: &title Do you want to use this address as a contact address?
+        heading: *title
```

Comment:
Question, what are the & and * sign doing in the title and heading, anything specific?